### PR TITLE
Morise decoder PR

### PR DESCRIPTION
**Pull request by rachaouldbabaali**
in this Pul Request we have done the following:

- [x] Added the decoder function for Morise code.
- [x] Installed linters.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,144 @@
...
+## 🔭Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank Microverse. for giving me this wonderful opportunity to work on this project and make me understand PostgreSQL deeply
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+
```
Comment (Gambit142) [inline README.md]: - [ ] The link to your MIT LICENSE is broken and displays a 404 error page when clicked. Kindly fix it by correcting the link. It should be `./LICENSE` and not `./LICENSE.md`. Please implement this change. 👍 😎

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @rachaouldbabaali @unorjikingsley 👋, 

Good job so far!

## Highlights
- All linter checks passed ✔️ 
- Created a method for decoding a Morse character ✔️ 
- Created a method for decoding a Morse word ✔️ 
- Created a method for decoding a Morse sentence ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by rachaouldbabaali** [comment]
Thank you @Gambit142 , I genuinely appreciate your time and effort in reviewing my work

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @rachaouldbabaali , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Well done fixing the link to the license. ✔ 🥳 🥳 🥳
- You successfully completed all the requirements.  🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rachaouldbabaali** [comment]
Thank you @ridwanediallo , I genuinely appreciate your time and effort in reviewing my work

**Pull request merged**